### PR TITLE
Dockerize app

### PR DESCRIPTION
**Pull request by vc0sta**
Hey, I've just created a Dockerfile and added instructions on how to execute it within a container.

As I'm not familiar with PHP tooling, I'm not sure if all those instructions on Dockerfile are really needed, please review it.

Also, I don't know if you want this feature added or not (feel free to reject this PR), but I like to run such tools in containers, so it would be great to have this option.

Very cool app btw

### CODE REVIEW

**Comment by nadrad** [comment]
Thanks, it seems interesting. 

Have you seen #11 ? Does your setup solve the clipboard issue in Docker?

**Comment by vc0sta** [comment]
application is running fine in RaspberryPi with no graphical environment installed on it.

Have not tested SSHing into it from my machine and pasting content yet.

**Comment by nadrad** [comment]
Can you check to see whether you can cut and paste nodes inside h-m-m? That's a normal function that depends on the clipboard.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+WORKDIR /app
+
+COPY ./h-m-m .
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+  libonig-dev xclip xsel wl-clipboard
+
+RUN docker-php-ext-install pcntl mbstring
```
Comment (Wojciechem) [inline Dockerfile:11]: could be merged with RUN above – 1 layer less

**Comment by jonas-w** [comment]
It seems to be working, except the clipboard support.

This would work, it will not sync with the hosts clipboard but would make it usable to copy and paste inside of the docker container:
https://github.com/asweigart/pyperclip/issues/161#issuecomment-1112942799

**Pull request merged**